syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

//定义消息登录类型
message GetFriendListsRequest
{
   uint32 userid = 1;
}

//定义消息响应消息
message GetFriendListsResponse
{
    ResultCode result = 1;
    repeated bytes friends = 2;
}

//好友模块
service FriendServiceRpc
{
    rpc GetFriendLists(GetFriendListsRequest) returns(GetFriendListsResponse);
    
}